### PR TITLE
Describe language specific style guide in l10n guide

### PR DESCRIPTION
**Pull request by gochist**
/cc @zacharysarah @raelga

### CODE REVIEW

File: content/en/docs/contribute/localization.md [hunk 1]
```diff
@@ -228,6 +228,12 @@ other = "ICH BIN..."
 
 Localizing site strings lets you customize site-wide text and features: for example, the legal copyright text in the footer on each page.
 
+### Language specific style guide and glossary
+
+Some l10n team has its own language specific style guide and glossary for the consistency. For example:
```
Comment (zacharysarah) [inline content/en/docs/contribute/localization.md]: > Some language teams have their own language-specific style guide and glossary. For example, see the [Korean Localization Guide](/ko/docs/contribute/localization_ko/).

**Comment by gochist** [comment]
@tengqm @zacharysarah Thank you! I addressed the review comment.

**Pull request merged**